netlogo-gui/src/main/swing/BrowserLauncher.scala

```diff
@@ -27,6 +27,10 @@ object BrowserLauncher {
       if (! opened)
         osSpecificBrowserRunner.getOrElse(browserNotFound()).apply(uri.toString)
     } catch {
+      val unableToOpenBrowserError = "We were unable to open a browser on your system.\n" +
+      "This error can be reported to bugs@ccl.northwestern.edu"
```

Comment:
This looks good. You can pull this out of the method and into the class and replace the string on L114 with `unableToOpenBrowserError`.

Reply:
Done!